src/intl/en/page-upgrades-vision.json

```diff
@@ -18,23 +18,23 @@
   "page-upgrades-vision-scalability-desc-3": "Sharding upgrades will spread the data storage requirements across the entire network, no longer requiring every node to hold 100% of the data. Although this doesn't directly address scaling the execution of transactions, this problem is being addressed directly by <a href=\"/layer-2/\">layer 2</a> rollup solutions.",
   "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit.",
   "page-upgrades-vision-security": "Security",
-  "page-upgrades-vision-security-desc": "The planned upgrades improve Ethereum's security against coordinated attacks, like a 51% attack. This is a type of attack where if someone controls the majority of the network they can force through fraudulent changes.",
-  "page-upgrades-vision-security-desc-3": "The transition to proof-of-stake means that the Ethereum protocol has greater disincentives against attack. This is because in proof-of-stake, the validators who secure the network must stake significant amounts of ETH into the protocol. If they try and attack the network, the protocol can automatically destroy their ETH.",
-  "page-upgrades-vision-security-desc-5": "This isn't possible in proof-of-work, where the best a protocol can do is force entities who secure the network (the miners) to lose mining rewards they would have otherwise earned. To achieve the equivalent effect in proof-of-work, the protocol would have to be able to destroy all of a miner's equipment if they try and cheat.",
-  "page-upgrades-vision-security-desc-5-link": "More on proof of work",
-  "page-upgrades-vision-security-desc-8": "Ethereum's security model also needs to change because of the introduction of sharding. The Beacon Chain will coordinate all of the validators who will be responsible for asserting that all data has been made available, but every node will no longer be required to hold the entire history of the chain. A new role is also anticipated, known as a dedicated block builder, who will work alongside block proposers (validators) for efficient and safe block production. Proof-of-stake is a prerequisite to sharding.",
+  "page-upgrades-vision-security-desc": "The planned upgrades improve Ethereum's security against coordinated attacks.",
+  "page-upgrades-vision-security-desc-3": "In proof-of-stake additional security comes from greater crypto-economic disincentives against attack. This is because in proof-of-stake, the validators who secure the network must stake significant amounts of ETH into the protocol. If they try and attack the network, the protocol can automatically destroy their ETH.",
```

Comment:
```suggestion
  "page-upgrades-vision-security-desc-3": "In proof-of-stake additional security comes from greater crypto-economic disincentives against attack. This is because, in proof-of-stake, the validators who secure the network must stake significant amounts of ETH into the protocol. If they try and attack the network, the protocol can automatically destroy their ETH.",
```